build-in-container/scripts/setup.sh

```diff
@@ -59,6 +59,36 @@ check_specs() {
     fi
 }
 
+# setup custom repo to install RPMs from
+setup_custom_repo() {
+    echo "------------ Setting up custom repo ------------"
+    if [[ ! -z "${RPM_repo}" ]]; then
+        echo "------------ RPM_repo is $RPM_repo ------------"
```

Comment:
Would it be better to just allow the user to specifcy a .repo file directly? Anyone who is using custom repos is very likely to already have a repo file. We can provide an example in the readme if needed.